src/content/self-hosted/manage/airgap.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: AirGap Networks
+description: Installing Okteto in AirGap Networks
+sidebar_label: AirGap Networks
+id: airgap
+---
+
+import variables from '../../variables.json';
+import CodeBlock from '@theme/CodeBlock';
+import TiersList from "@theme/TiersList";
+
+# AirGap Networks<TiersList tiers="Enterprise Self-Hosted" />
+
+Okteto works on AirGap Networks, but it requires custom configuration.
+This guide has been crafted to help you overcome these hurdles, but we encourage you to [talk to us](https://www.okteto.com/get-demo/) to guide you during the installation.
+
+
+## Step 1: Disable Okteto Telemetry
+
+By default, Okteto sends [telemetry data](self-hosted/helm-configuration.mdx#telemetry) to help us understand usage patterns and prioritize improvements
+In air-gapped environments, you can disable telemetry by adding the following to your Okteto Helm configuration:
+
+```yaml
+telemetry:
+  enabled: false
+```
+
+Disabling telemetry will automatically stop Okteto CLI analytics for all developers.
+
+## Step 2: Host Images in your Private Registry
+
+By default, all the images needed to deploy the Okteto Platform and the images used by the Okteto CLI are hosted in DockerHub.
+In air-gapped environments, you’ll need to host these images in your private registry.
```

Comment:
I was planning to add where to find all the images needed. Do you think is too much information for the guide? I mean, I wouldn't add every image we need, but I would add where to find them. 

For example, in the Chart Artifactory, in the values file, in the dependencies section for the nginx ...

Reply:
I know we added all of them as a list later here, but that means that every time we add a new one we need to update this guide. For me its better if we teach them how to fish instead giving them fish :)

Reply:
Those are our own images, they don't change that much. We can list them for now and iterate if needed